Objects/Converters/ConverterRhinoGh/ConverterRhinoGh/ConverterRhinoGh.Units.cs

```diff
@@ -13,13 +13,15 @@ namespace Objects.Converter.RhinoGh
   public partial class ConverterRhinoGh
   {
     private string _modelUnits;
+    
     public string ModelUnits
     {
       get
       {
-        if (string.IsNullOrEmpty(_modelUnits))
+        var unitToSpeckle = UnitToSpeckle(Doc.ModelUnitSystem);
```

Comment:
Hey Alan,
I have two comments on this approach :
- we don't need to cache the _modelUnits every time, so it could be simplified
- we still need to make sure an up-to-date Doc is passed to the converters every time before converting